mitosheet/mitosheet/public/v3/formatting.py

```diff
@@ -50,6 +49,16 @@ def get_conditional_format_rule(
         formula = [f'{cell_range}{comparison}DATEVALUE("{filter_value}")']
     elif comparison is not None:
         formula = [f'{cell_range}{comparison}{filter_value}']
+    elif filter_condition == FC_NUMBER_EXACTLY:
+        # Sometimes, users will enter a string for a number filter -- because it is just
+        # displayed as an = on the frontend. This is a reasonably thing for them
+        # to do -- so we take special care in this case to wrap the filter value in
+        # quotes
+        try:
+            float(filter_value)
+            formula = [f'{cell_range}={filter_value}']
+        except ValueError:
+            formula = [f'{cell_range}="{filter_value}"']
```

Comment:
Can we add a test for this that actually generates the Excel file and then tests that it has conditional formatting applied to it?

Reply:
We cannot test this with python, unfortunately.